modules/generator/processor/localblocks/config.go

```diff
@@ -0,0 +1,39 @@
+package localblocks
+
+import (
+	"flag"
+	"time"
+
+	"github.com/grafana/tempo/tempodb"
+	"github.com/grafana/tempo/tempodb/encoding"
+	"github.com/grafana/tempo/tempodb/encoding/common"
+)
+
+const (
+	Name = "local-blocks"
+)
+
+type Config struct {
+	Block                *common.BlockConfig   `yaml:"block"`
+	Search               *tempodb.SearchConfig `yaml:"search"`
+	FlushCheckPeriod     time.Duration         `yaml:"flush_check_period"`
+	MaxTraceIdle         time.Duration         `yaml:"trace_idle_period"`
```

Comment:
Nit: do we want to align the field tag with the name?  This is the only one that doesn't match.

Reply:
Good catch, I copied from https://github.com/grafana/tempo/blob/main/modules/ingester/config.go#L23.